toolbox/cluster.py

```diff
@@ -54,3 +56,24 @@ def capture_environment():
             image: The image to upgrade the cluster to
         """
         return PlaybookRun("capture_environment")
+
+    @staticmethod
+    def wait_for_alert(alert_name, alert_active: bool):
+        """
+        Wait for an alert to be active or inactive.
+
+        Args:
+            alert_name: The name of the alert to wait for
+            alert_active: A boolean telling if the alert should be active or not (true|false)
+        """
+
+        if alert_active not in ("true", "false"):
```

Comment:
Yikes. I must say I'm incredibly disappointed in how much non-strict is Fire with regards to things like this. https://github.com/google/python-fire/issues/221